Source/WebInspectorUI/UserInterface/Views/WebInspectorExtensionTabContentView.js

```diff
@@ -119,13 +124,6 @@ WI.WebInspectorExtensionTabContentView = class WebInspectorExtensionTabContentVi
 
     _extensionFrameDidLoad()
     {
-        // Bounce from the initial empty page to the requested sourceURL.
-        if (!this._iframeFinishedInitialLoad) {
-            this._iframeFinishedInitialLoad = true;
-            WI.sharedApp.extensionController.evaluateScriptInExtensionTab(this._extensionTabID, `document.location.replace("${this._sourceURL}");`);
-            return;
-        }
-
         // Signal that the page is available since we already bounced to the requested page.
         if (!this._whenPageAvailablePromise.settled)
```

Comment:
I think we need to reset/recreate this inside `set iframeURL`.  If we leave it as-is, then it probably would negatively impact `WI.WebInspectorExtensionController.prototype.showExtensionTab` because it'd always be resolved after the first load.

Reply:
Always being resolved after the first load is by design, we just care if a page ever loaded for the clients of this promise.